app/Http/Controllers/CrudController.php

```diff
@@ -358,6 +359,12 @@ public function crudStore(Request $request, bool $redirectToCreated = false)
     {
         $this->authorize('create', $this->model);
 
+        $request->validate([
```

Comment:
This should be in the StoreCreature, StoreCharacter etc rules, since not all entities have attributes. That way it will also be done in all the APIs